qiskit_aer/quantum_info/states/aer_statevector.py

```diff
@@ -184,27 +184,36 @@ def _aer_evolve_circuit(aer_state, circuit, qubits):
     @staticmethod
     def _aer_evolve_instruction(aer_state, inst, qubits):
         """Apply instruction into aer_state"""
+
         params = inst.params
+
         if inst.name in ['u', 'u3']:
```

Comment:
Given other conversations, should `cu` be in this list? It doesn't seem to be (maybe I can't spot it), and I'm guessing it's not tested?